lte/gateway/c/session_manager/SessionStateEnforcer.cpp

```diff
@@ -403,7 +402,26 @@ void SessionStateEnforcer::m5g_move_to_inactive_state(
   /* Call for all rules to be de-associated from session
    * and inform to UPF
    */
-  m5g_pdr_rules_change_and_update_upf(imsi, session, PdrState::IDLE);
+  m5g_pdr_rules_change_and_update_upf(session, PdrState::IDLE);
+  return;
+}
+
+void SessionStateEnforcer::m5g_move_to_active_state(
+    std::unique_ptr<SessionState>& session, SetSmNotificationContext notif,
+    SessionStateUpdateCriteria* session_uc) {
+  bool gnb_teid_get = false;
+  bool upf_teid_get = false;

```

Comment:
```suggestion
  bool get_gnb_teid = false;
  bool get_upf_teid = false;

```

Reply:
Updated the identifier names as suggested
gnb_teid_get -> get_gnb_teid
upf_teid_get -> get_upf_teid